doc/whats_new/v0.20.rst

```diff
@@ -422,6 +422,15 @@ Linear, kernelized and related models
   better for unscaled features. :issue:`8361` by :user:`Gaurav Dhingra <gxyd>`
   and :user:`Ting Neo <neokt>`.
 
+Feature Extraction
+
+- Deprecate ``stop_words='english'`` parameter in 
+  :class:`feature_extraction.text.CountVectorizer`, 
+  :class:`feature_extraction.text.TfidfVectorizer`, 
+  :class:`feature_extraction.text.HashingVectorizer` as the current list is 
+  broken, leading to unexpected results. :issue:`10735` 
```

Comment:
While the general feeling is that "current list is broken" the main point from https://github.com/scikit-learn/scikit-learn/issues/10735#issuecomment-369614044 IMO is "There is no one true authoritative English stopword list". Saying it's broken means it could be fixed while the situation is more complicated than that.  Maybe,
```
as the current list is controversial and may lead to unexpected results.
```
?

Reply:
Sounds good to me. Sure will do

Reply:
Done